.gitmodules

```diff
@@ -18,3 +18,25 @@
   path = stochastic_physics
   url = https://github.com/noaa-psd/stochastic_physics
   branch = master
+[submodule "CMakeModules"]
+  path = CMakeModules
+  url = https://github.com/NOAA-EMC/CMakeModules
+  branch = develop
+[submodule "CMEPS"]
+  path = CMEPS-interface/CMEPS
+  url = https://github.com/NOAA-EMC/CMEPS.git
+  branch = emc/develop
+[submodule "MOM6"]
+  path = MOM6-interface/MOM6
+  url = https://github.com/NOAA-EMC/MOM6
+  branch = dev/emc
+[submodule "CICE"]
```

Comment:
would you please  merge to the latest ufs-weather develop branch?

Reply:
Yes.